scripts/zones/RuAun_Gardens/mobs/Sprinkler.lua

```diff
@@ -6,11 +6,23 @@ require("scripts/globals/regimes")
 -----------------------------------
 local entity = {}
 
+entity.onMobMagicPrepare = function(mob, target, spellId)
+    local cast = math.random()
+
+    if cast >= 0.9 then
+        return 212 -- Burst
+    elseif cast >= 0.7 then
+        return 167 -- Thunder IV
+    elseif cast >= 0.5 then
+        return 196 -- Thundaga III
+    else
+        return 166 -- Thunder III
```

Comment:
Source for these probabilities? (it's been 15+ years since I paid any attention to Sky, so I don't know)

Reply:
this is a block of code formerly removed from the project when we found out it was bogus.